src/lib/core/theming/_palette.scss

```diff
@@ -660,6 +661,7 @@ $md-dark-theme-background: (
   hover:      rgba(white, 0.04), // TODO(kara): check style with Material Design UX
   card:       map_get($md-grey, 800),
   dialog:     map_get($md-grey, 800),
+  sidenav:    #303030,
```

Comment:
You should just use the existing `background` key. (that's where I got the #303030 from)